zulip_bots/zulip_bots/bots/mention/test_mention.py

```diff
@@ -0,0 +1,31 @@
+from zulip_bots.test_lib import BotTestCase
+
+
+class TestMentionBot(BotTestCase):
+    bot_name = "mention"
+
+    def test_bot_responds_to_empty_message(self) -> None:
+        with self.mock_config_info({'access_token': 'MjNjMTIwMjIxNmRlYWE4OWE5Yzg3ZTkzOWJlNDRhZWM0OGZmZWVmMzZmY2Q3ODQ3YjYyYjg4OTJhZDljMDgyOQ'}):
```

Comment:
We try to avoid tests that access the internet, because they tend to be unreliable. Therefore, we use the `mock_http_conversation` function to mock any http traffic during a bot test. That's what the fixtures are created for. Once you use that function, you can also use a shorter API key, e.g. "12345678".

Reply:
I couldn't find a way to test online queries as the response from the Mention API is refreshed every time. These tests are offline in nature and return before the internet is accessed by the bot. I have added comments.

Reply:
I have added the files to `run-mypy` but upon run, I get the error that type `response`  is not index able. I have submitted without fixing this error, frankly because I couldn't  figure out how to.